docs/concepts.rst

```diff
@@ -0,0 +1,28 @@
+============
+Concepts
+============
+
+.. _named_table:
+
+Named Tables
```

Comment:
If a user did something along the lines:
```
table = Table(conn_id="sample_conn_id")
table.temp = False
```
The table would be "unnamed" (the Python SDK would be generating its name), but a persistent table. Therefore, I think it is safer to call it Persistent table as opposed to Named.

Reply:
I missed this case, will change accordingly.